kolibri/utils/cli.py

```diff
@@ -87,11 +88,74 @@
 
 logger = logging.getLogger(__name__)
 
+KOLIBRI_HOME = os.environ['KOLIBRI_HOME']
+VERSION_FILE = os.path.join(KOLIBRI_HOME, '.data_version')
+
+
+class PluginDoesNotExist(Exception):
+    """
+    This exception is local to the CLI environment in case actions are performed
+    on a plugin that cannot be loaded.
+    """
+    pass
+
+
+def _first_run():
+    """
+    Called once at least. Will not run if the .kolibri/.version file is
+    found.
+    """
+    if os.path.exists(VERSION_FILE):
+        logger.error("_first_run() called, but Kolibri is already initialized.")
+        return
+    logger.info("Kolibri running for the first time.")
+    logger.info(
+        "We don't yet use pre-migrated database seeds, so you're going to have "
+        "to wait a bit while we create a blank database...\n\n"
+    )
+
+    django.setup()
+
+    call_command("migrate")
+
+    from kolibri.core.settings import WELL_KNOWN_PLUGINS
```

Comment:
Can we circumvent having to enable core plugins this way? For instance, wouldn't it be simpler to just add the core plugin modules to INSTALLED_APPS?